### PR TITLE
Bug/v1-195 | Long name of course title on Employee Profile page for Manager

### PR DESCRIPTION
**Pull request by PechurinaLena**
1. Cut long name of course title on Employee Profile page for Manager
2. Added hover effect to show full course title
3. Refactored code: added new container for mapped CourseItem 
4. Improving styles for hide last divider

### CODE REVIEW

File: frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/EmployeeCourses/EmployeeCoursesList/CourseItem/CourseItem.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { FC } from 'react';
+
+import ProgressBar from 'components/ProgressBar';
+
+import { Size } from 'enums/sizes';
+import { setFirstLetterUppercase } from 'utils/helpers/setFirstLetterUppercase';
```
Comment (irina-sinitsyna) [inline frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/EmployeeCourses/EmployeeCoursesList/CourseItem/CourseItem.tsx:6]: fix imports please

File: frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/EmployeeCourses/EmployeeCoursesList/CourseItem/CourseItemContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { FC, useState } from 'react';
+
+import { convertEmployeeCourseProgress } from 'utils/helpers/convertCourseStatusToProgress';
+
+import CourseItem from './CourseItem';
+import { ICourseItemContainerProps } from './types';
+
+const CourseItemContainer: FC<ICourseItemContainerProps> = ({ title, status }) => {
+  const [isShown, setIsShown] = useState(false);
```
Comment (irina-sinitsyna) [inline frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/EmployeeCourses/EmployeeCoursesList/CourseItem/CourseItemContainer.tsx:9]: please make the name more descriptive, like isCourseInfoShown

File: frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/EmployeeCourses/EmployeeCoursesList/CourseItem/styled.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  fontSize: '12px',
+  backgroundColor: '#000',
+  color: '#FFF',
+  overflow: 'inherit',
+  textOverflow: 'unset',
+  boxShadow: '0 0 10px rgba(0,0,0,0.5)',
+});
+
+export const CourseStatus = styled(Typography)({
+  color: '#b6b6b6',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/EmployeeCourses/EmployeeCoursesList/CourseItem/styled.ts:70]: uppercase

File: frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/EmployeeCourses/EmployeeCoursesList/EmployeeCoursesList.tsx [hunk 4]
```diff
@@ -1,40 +1,22 @@
-import React from 'react';
+import { FC } from 'react';
 
-import ProgressBar from 'components/ProgressBar';
 import NoContent from 'components/NoContent';
 import { NO_USER_COURSES } from 'constants/messages';
-import { IEmployeeCoursesList } from 'types/employee';
-import { setFirstLetterUppercase } from 'utils/helpers/setFirstLetterUppercase';
-import { convertEmployeeCourseProgress } from 'utils/helpers/convertCourseStatusToProgress';
+
```
Comment (irina-sinitsyna) [inline frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/EmployeeCourses/EmployeeCoursesList/EmployeeCoursesList.tsx:5]: imports order

File: frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/EmployeeCourses/EmployeeCoursesList/EmployeeCoursesList.tsx [hunk 5]
```diff
@@ -1,40 +1,22 @@
...
 
-const EmployeeCoursesList: React.FC<IEmployeeCoursesList> = ({ courses }) => (
+const EmployeeCoursesList: FC<IEmployeeCoursesList> = ({ courses }) => (
   <CoursesList>
     {courses?.length ? (
-      courses.map((course, id, coursesArray) => {
-        const isDividerVisible = id < coursesArray.length - 1;
-        const { progressValue, progressVariant } = convertEmployeeCourseProgress(course.status);
+      courses.map((course) => {
         return (
```
Comment (irina-sinitsyna) [inline frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/EmployeeCourses/EmployeeCoursesList/EmployeeCoursesList.tsx:16]: Do we still need return here?

**Review by ostapdans99 — APPROVED** [review]

**Pull request merged**